synapse/handlers/deactivate_account.py

```diff
@@ -120,6 +120,11 @@ async def deactivate_account(
 
         await self.store.user_set_password_hash(user_id, None)
 
+        # Most of the pushers will have been deleted when we logged out the
```

Comment:
is this true?

Reply:
Ugh, the logic looks to be broken when using workers.

Reply:
This has now been fixed